.scrutinizer.yml

```diff
@@ -3,7 +3,7 @@ inherit: true
 build:
   environment:
     php:
-      version: 8.2
+      version: 8.4
```

Comment:
It seems that PHP 8.4 won't build on scrutinizer: https://scrutinizer-ci.com/g/jakzal/toolbox/inspections/ed3fed18-9fb1-4dbd-80a8-2f578500089b